gmso/formats/networkx.py

```diff
@@ -0,0 +1,953 @@
+                if len(node.label) > 12:
+                    node.label = "".join([line + "\n" for line in node.label.split()])
+        labels[node] = node.label
+
+    return labels
+
+
+def show_parameter_values(topology, list_of_params, checkbox):
+    if checkbox:
+        try:
+            report_parameter_expression(topology, list_of_params[0])
+        except AttributeError:
+            print("There are no values for the parameter expression")
+    else:
+        # TODO: Should be able to remove this blank print statement so that deselecting the
+        # checkbox removes the listed parameters.
+        print("")
+
+    return
+
+
+def report_parameter_expression(topology, param):
+    # return nicely printed parameters for a given edge.
+    if len(param) == 4:
+        try:
+            for dihedral in list(topology.dihedrals):
+                if dihedral.connection_members == (
+                    param[0],
+                    param[1],
+                    param[2],
+                    param[3],
+                ):
+                    print(dihedral.dihedral_type.expression, "\n")
+                    print("{:<12} {:<15}".format("Parameter", "Value"))
+                    for k, v in dihedral.dihedral_type.parameters.items():
+                        print("{:<12} {:<15}".format(k, v))
+        except AttributeError:
+            print("Dihedral not typed")
+    elif len(param) == 3:
+        try:
+            for angle in list(topology.angles):
+                if angle.connection_members == (param[0], param[1], param[2]):
+                    print(angle.angle_type.expression, "\n")
+                    print("{:<12} {:<15}".format("Parameter", "Value"))
+                    for k, v in angle.angle_type.parameters.items():
+                        print("{:<12} {:<15}".format(k, v))
+        except AttributeError:
+            print("Angle not typed")
+    else:
+        raise ValueError(
+            "Parameters are not proper angles or dihedrals. Connection members are missing"
+        )
+
+    return
+
+
+def get_edges(networkx_graph, atom_name1, atom_name2):
+
+    # Create a list of the edges that have been selected corresponding to the selected atom_name1
+    # and atom_name2. If both are None, then select edges that are missing bond types.
+    list_of_edges = []
+    list_of_bonds = []
+    # Check for whether you want to plot bonds based on the atom types, or their node labels.
+    try:
+        [node.atom_type.name for node in networkx_graph.nodes]
+    except AttributeError:
+        return print("Atomtypes are missing, so no bond types can be selected")
+    mia_bond_flag = 0
+    if atom_name1 and atom_name2:
+        for edge in list(networkx_graph.edges):
+            if (
+                edge[0].name == atom_name1
+                and edge[1].name == atom_name2
+                or edge[0].name == atom_name2
+                and edge[1].name == atom_name1
```

Comment:
Might be nicer to clarify precedence in this case not to get confused later. Although, and takes precedence over or in this case.
```suggestion
                (edge[0].name == atom_name1
                and edge[1].name == atom_name2)
                or (edge[0].name == atom_name2
                and edge[1].name == atom_name1)
```